datastore/transaction.go

```diff
@@ -110,17 +110,22 @@ func (c *Client) NewTransaction(ctx context.Context, opts ...TransactionOption)
 	return c.newTransaction(ctx, newTransactionSettings(opts))
 }
 
-func (c *Client) newTransaction(ctx context.Context, s *transactionSettings) (*Transaction, error) {
+func (c *Client) newTransaction(ctx context.Context, s *transactionSettings) (_ *Transaction, err error) {
 	req := &pb.BeginTransactionRequest{ProjectId: c.dataset}
 	if s.readOnly {
 		req.TransactionOptions = &pb.TransactionOptions{
 			Mode: &pb.TransactionOptions_ReadOnly_{ReadOnly: &pb.TransactionOptions_ReadOnly{}},
 		}
-	} else if s.prevID != nil {
-		req.TransactionOptions = &pb.TransactionOptions{
-			Mode: &pb.TransactionOptions_ReadWrite_{ReadWrite: &pb.TransactionOptions_ReadWrite{
-				PreviousTransaction: s.prevID,
-			}},
+	} else {
+		ctx = trace.StartSpan(ctx, "cloud.google.com/go/datastore.Transaction.ReadWriteTransaction")
```

Comment:
Why is the span not started for readOnly? Can you add a comment to clarify?

Reply:
I don't really understand why there wouldn't be a span for both transaction types. @benwhitehead can you follow up to confirm?

Reply:
@tritone Is correct, we should cover both cases I missed adding readOnly to the list when I was grepping for stuff apologies, I'll add it to the list in the doc.

Reply:
All right, I added a span for `readOnly` also.

Reply:
Yes, that confused me too a bit